tools/go.mod

```diff
@@ -4,7 +4,7 @@ go 1.22.4
 
 require (
 	github.com/golang/mock v1.6.0
-	github.com/golangci/golangci-lint v1.59.1
+	github.com/golangci/golangci-lint v1.60.1
```

Comment:
will need to fix some linter violations before merge. a few of the linters got new rules that would cause our lint CI to go red after merge.